### ISSUE TITLE
Synthetic Controller not working despite Developer Mode on

### ISSUE THREAD

**Issue by PoretJon** [InvoxiPlayGames/FestivalInstrumentMapper#13]
As the title says, using Windows 10 with a Santroller guitar, getting error 80010111 with failed to create synthetic controller.
![rror](https://github.com/InvoxiPlayGames/FestivalInstrumentMapper/assets/90916905/80cabc80-2b51-4afa-8521-f2f3edf5c577)

**Comment by TheNathannator** [comment]
hmm...

![image](https://github.com/InvoxiPlayGames/FestivalInstrumentMapper/assets/29052821/241614e4-b495-4cc9-b9f9-3e814c140997)

**Comment by PoretJon** [comment]
![seconderror](https://github.com/InvoxiPlayGames/FestivalInstrumentMapper/assets/90916905/e0bddb58-7f38-469e-b6ea-60503b131d6f)
was messing around and tried again, now im getting a different error? not sure if this is an issue with my windows install or something else now

**Comment by Yeldaarb** [comment]
I'm also getting this same error using my wiitar with wiitar thing. my PC is up to date and I have developer mode on as well.

**Comment by InvoxiPlayGames** [comment]
are Xbox Gaming Services and all the other associated apps and services installed on your computer?

**Comment by PoretJon** [comment]
yes, although looking at services.msc my xbox accessory management service is stuck on starting, maybe thats the problem?

**Comment by Yeldaarb** [comment]
Yes they are, I even ran the xbox gaming services repair tool from microsoft, but that didn't fix the issue

**Comment by PoretJon** [comment]
just tried beta 2, getting a mix of both errors with no real pattern to them

**Comment by Yeldaarb** [comment]
same here, but only still getting error 80080005

**Comment by ThatOneLittleNoob** [comment]
also same here I have the same error code 8008005
![FestivalInstrumentMapper_B5jZrPxyfG](https://github.com/InvoxiPlayGames/FestivalInstrumentMapper/assets/116435678/fd6bd809-0023-43e3-89d1-7243c6de17dc)

**Comment by CoreParadox** [comment]
> yes, although looking at services.msc my xbox accessory management service is stuck on starting, maybe thats the problem?

I also have this problem, looking at event-viewer logs shows this:

The server Windows.Internal.Gaming.SWDDeviceStatics did not register with DCOM within the required timeout.

Just normal windows 11 pro here

**Comment by SMHUNTERSM** [comment]
i have the same problem with a wiiguitar before the error the program freezes and after a few seconds it starts working again but with the error 80080005 could you solve it?
